### ISSUE TITLE
[Question] Programmatically Clear Entire Cache

### ISSUE THREAD

**Issue by jakemumu** [DmitryTsepelev/graphql-ruby-fragment_cache#93]
Hey -- 

Wondering if there is a way to for the entire cache to clear for all queries as opposed to by query by query. 

Is there a way to clear the cache of all cached fragments? Such as Rails.cache.clear?

**Comment by DmitryTsepelev** [comment]
Hey @jakemumu! It depends on the storage you use: for instance in Redis you can remove all keys starting with `graphql/`

**Comment by jakemumu** [comment]
For sure -- I noticed however when using no memory store in local development, it defaults to its internal memory mechanism. I haven't seen a way to disable or clear this one when working locally? Apologies if it's there.

**Comment by jakemumu** [comment]
Amazing library by the way! So helpful over caching entire queries.

**Comment by DmitryTsepelev** [comment]
You can clear a memory one as well (`.clear`, [here](https://github.com/DmitryTsepelev/graphql-ruby-fragment_cache/blob/master/lib/graphql/fragment_cache/memory_store.rb#L56)) but I'm not sure if it's possible to connect to the running process from the console. Memory story is just a hash in the memory of a running process 🙂

**Comment by DmitryTsepelev** [comment]
BTW, what problem are you solving? Maybe we can try an alternative solution

**Comment by jakemumu** [comment]
I think I've made an idiotic mistake 0.o -- I have some bulk edits I've been making and checking on the API while working locally, I noticed I wasn't able to clear the cache, but I've just realized... simply restarting the rails server will dump the in memory cache! derp! -- totally can close this, idk how I didn't think of that

**Comment by DmitryTsepelev** [comment]
Indeed, restart will definitely drop the cache 😅But in production we should make sure to prefer something that _persists_, otherwise we'll have weird performance while cache is warming. Please ping me if you need anything else!
